internal/fs/ea_windows.go

```diff
@@ -298,3 +299,20 @@ func PathSupportsExtendedAttributes(path string) (supported bool, err error) {
 	supported = (fileSystemFlags & windows.FILE_SUPPORTS_EXTENDED_ATTRIBUTES) != 0
 	return supported, nil
 }
+
+// GetVolumePathName returns the volume path name for the given path.
+func GetVolumePathName(path string) (volumeName string, err error) {
```

Comment:
Please add a minimal test to verify that the function call wrapper works as expected.

Reply:
done